rac-gcp-deploy/orb.yml

```diff
@@ -20,7 +20,8 @@ orbs:
 
 executors:
   machine:
-    machine: true
+    machine:
```

Comment:
Is this basically redefining a built in called "machine"? Just trying to work out how to interpret this nesting

Reply:
`machine` is the default CircleCi executor which uses a default image. That default image is deprecated and we need to specify an up to date image.

Reply:
I might misinterpreted your question. 
We are just naming an executor `machine` which _uses_ the default  `machine` executor. We do this so we are able to reference to it outside of the orb

Reply:
@jens can you approve this or is there anything you would like to change?

Reply:
Nope that's fine I just missed your replies 🙈